### PR TITLE
Run sanity tests

### PR DESCRIPTION
**Pull request by vvarg229**
Before merging, merge this PR: https://github.com/nspcc-dev/neofs-testcases/pull/560

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -58,6 +58,7 @@ jobs:
 
 ################################################################
       - name: Build neofs-node (neofs-cli and neofs-adm)
+        timeout-minutes: 30
```
Comment (roman-khimov) [inline .github/workflows/run-tests.yml:62]: I'd be a bit more aggressive here, 30m is too much to build some Go binaries.

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -203,6 +212,7 @@ jobs:
           allure_history: allure-history
 
       - name: Deploy report to Github Pages
+        timeout-minutes: 120
```
Comment (roman-khimov) [inline .github/workflows/run-tests.yml:216]: 2h to push git changes to GH?

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - support/**
```
Comment (roman-khimov) [inline .github/workflows/run-tests.yml:7]: NACK, not needed.

File: .github/workflows/run-tests.yml (hunk 2 above)
Comment (vvarg229) [reply .github/workflows/run-tests.yml:216]: > 2h to push git changes to GH?

Yes, that can happen.
I did my worst-case testing by rebasing, deleting files, pushing with force and e.t.c
I also did it all on my personal runner with a slow internet connection.

Here is the slowest result:
https://github.com/vvarg229/neofs-node/actions/runs/4895100485/jobs/8740213411
`1h 6m 2s
`
I am surprised and shocked by this result, so I will try to rework the publication of the allure reports to a static web page as soon as possible.

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (vvarg229) [reply .github/workflows/run-tests.yml]: Done

File: .github/workflows/run-tests.yml (hunk 3 above)
Comment (vvarg229) [reply .github/workflows/run-tests.yml]: Done

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**